### ISSUE TITLE
[feature request] Store date

### ISSUE THREAD

**Issue by mkounkel** [comictagger/comictagger#438]
It would be nice if, instead of the cover date, it would be possible to have it pull the store date, may be with a configurable option of which one is being preferred by the user.

**Comment by lordwelch** [comment]
Good idea, so what would you expect if there is a cover date but no store date?

**Comment by mkounkel** [comment]
May be also configurable - either defaulting to the cover date or leave blank, this way people would be able to chose.

**Comment by lordwelch** [comment]
Can you test #450 and see how well it works for you?
[downloading packages from pull requests](https://github.com/comictagger/comictagger/wiki/downloading-packages-from-pull-requests)

**Comment by mkounkel** [comment]
I don't think it compiled correctly - since it doesn't seem to have produced any output files to download.

**Comment by lordwelch** [comment]
It appears that Chocolatey (a windows package manager) was down earlier the build should be up shortly.

edit: Never mind it appears some of my tests are still failing...

**Comment by lordwelch** [comment]
@mkounkel Ok, now it has built successfully

**Comment by mkounkel** [comment]
Thanks!
It's downloading the new date, but I'm not sure if it's writing it or not - if I close it, and read the same file that it has just written, all of the information regarding pubdate disappears, and looking at the raw tags, it doesn't appear anywhere in there.

Otherwise, I think it could work - I'm using it with conjunction with https://github.com/dickloraine/EmbedComicMetadata plugin for Calibre, and as long as the existing tags don't change, but the new ones get added and it is possible to figure out what they are, I think I should be able to set it up to extract the relevant info.

**Comment by lordwelch** [comment]
So all of the metadata formats currently supported do not have support for both the cover date and the publish or store date. For this PR I made a setting to choose which date will be read and saved to metadata that does not support both dates. You will need to set this to publish to get what you want.
<img width="850" alt="Screenshot 2023-04-15 at 10 26 00 AM" src="https://user-images.githubusercontent.com/7547075/232237938-eda05016-5d31-45c9-85cd-5d68cba0bdce.png">

**Comment by mkounkel** [comment]
Ah, I didn't notice it in the settings at first. But, hmm, nope, no matter what it is set to, it is still writing out the cover date - this setting seems to only control where that date is later being read to, if it is opened again.
<img width="116" alt="image" src="https://user-images.githubusercontent.com/6738789/232339282-149fa4a7-001c-4da9-b732-d4dc8e63d672.png">
<img width="112" alt="image" src="https://user-images.githubusercontent.com/6738789/232339333-0a25ba74-6f63-4ea3-87bb-310a444fb29f.png">

**Comment by mkounkel** [comment]
Spit-balling a couple of ideas - if it seems like there is only one possible way to store the date and the cover date is treated as the default - and if it is impossible to create a dummy tag with the second date that would comply with the comicinfo.xml standard - may be the simplest thing to do would be to just throw it into the notes, with some sort of a convenient delimiter for splitting? This could be a catch-all field for all of the other metadata stored by comicvine for an issue that do not have a dedicated tag. This way, the standard itself is preserved, but if someone wants to have this info for a database it is in principle accessible and doesn't need to be redownloaded to access.
